instrumentation/opentelemetry-instrumentation-openai/pyproject.toml

```diff
@@ -0,0 +1,55 @@
+[build-system]
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+
+[project]
+name = "opentelemetry-instrumentation-openai-v2"
+dynamic = ["version"]
+description = "OpenTelemetry Official OpenAI instrumentation"
+readme = "README.rst"
+license = "Apache-2.0"
+requires-python = ">=3.8"
+authors = [
+  { name = "OpenTelemetry Authors", email = "cncf-opentelemetry-contributors@lists.cncf.io" },
+]
+classifiers = [
+  "Development Status :: 4 - Beta",
+  "Intended Audience :: Developers",
+  "License :: OSI Approved :: Apache Software License",
+  "Programming Language :: Python",
+  "Programming Language :: Python :: 3",
+  "Programming Language :: Python :: 3.8",
+  "Programming Language :: Python :: 3.9",
+  "Programming Language :: Python :: 3.10",
+  "Programming Language :: Python :: 3.11",
+  "Programming Language :: Python :: 3.12",
+]
+dependencies = [
+  "opentelemetry-api ~= 1.12",
+  "opentelemetry-instrumentation ~= 0.48b0",
+  "opentelemetry-semantic-conventions ~= 0.48b0"
+
+]
+
+[project.optional-dependencies]
+instruments = [
+  "openai >= 0.27.0",
```

Comment:
Was this tested on openai v0.27.0? According to the test-requirements.txt, this was tested on openai==1.37.1 so the min version should be that no?

Reply:
yes exactly, will fix now